.github/maintainers_guide.md

```diff
@@ -53,22 +53,25 @@ do so after you are done working in this project. To come back to development
 work for this project again in the future, `cd` into this project directory and
 run `source env_3.9.6/bin/activate` again.
 
-The last step is to install this project's dependencies; to do so, check out [how
-we configure GitHub Actions to install dependencies for this project for use in
-our continuous integration](https://github.com/slackapi/python-slack-sdk/blob/main/.github/workflows/ci-build.yml#L26-L30). You can also run `./scripts/run_validation.sh` to install the dependencies and run the unit tests in one command!
+The last step is to install this project's dependencies and run all unit tests; to do so, you can run
+```bash
+$ ./scripts/run_validation.sh 
+```
 
+Also check out [how
+we configure GitHub Actions to install dependencies for this project for use in
+our continuous integration](https://github.com/slackapi/python-slack-sdk/blob/main/.github/workflows/ci-build.yml#L26-L30).
 ## Tasks
 
 ### Testing (Unit Tests)
 
 When you make changes to this SDK, please write unit tests verifying if the changes work as you expected. You can easily run all the tests by running the command. The `validate` command runs Flake8 (static code analyzer), Black (code formatter), and unit tests in the `tests` directory for you.
```

Comment:
I think we need updates to this sentence too as it still references the old `python setup.py validate` command. Also "You can easily run all the tests by running the command" - "the command" is ambiguous at this point; perhaps better something like "You can easily run all the tests by running the script below".

Reply:
Maybe we should remove these lines entirely and let people look into the GitHub actions to find what dependencies get installed. Alternatively they can also go through the `./scripts/run_validation.sh` to view what gets installed?

Reply:
I'd just change this to something like:

```suggestion
When you make changes to this SDK, please write unit tests verifying if the changes work as you expected. You can easily run all the tests and formatting/linter by running:
```

Reply:
Agreed, I think I miss-understood what lines we were referring to.